stubs/pynput/@tests/stubtest_allowlist.txt

```diff
@@ -0,0 +1,9 @@
+pynput
+pynput._info
+pynput._util.RESOLUTIONS
+pynput.keyboard
+pynput.keyboard._base
+pynput.keyboard._dummy
+pynput.mouse
+pynput.mouse._base
+pynput.mouse._dummy
```

Comment:
Why are all of those on the stubtest allowlist?

Reply:
stubtest fails to import `pynput` since there's no display server running (e.g. x11)

Reply:
I see. We should probably re-introduce the flag to disable stubtest in `METADATA.toml` we had for a short time. I will send a PR tomorrow.

Reply:
We could run stubtest under xvfb, which emulates an x11 display. For example: https://github.com/Akuli/porcupine/blob/1ad0d5fa8d944ef66a0dbb7e0100da3a13aebfda/.github/workflows/check.yml#L60-L62

Reply:
Am I missing something? stubtest does not appear to fail to import pynput in CI. In fact, if it was failing, you wouldn't see any errors.

Reply:
@hauntsaninja Here is the failing CI test in the first commit:

https://github.com/python/typeshed/runs/5163416378
```
error: pynput failed to import: this platform is not supported: ('failed to acquire X connection: Bad display name ""', DisplayNameError(''))
```

I added the allowlist in the commit after it.

Reply:
Oh, I see how I confused myself. Sorry about that, thanks!